### ISSUE TITLE
GPU computation is much faster than CPU

### ISSUE THREAD

**Issue by Ecofriendlyhooney** [Ecofriendlyhooney/tensorflow_deep_learning#8]
![GPU_modeling_speed](https://user-images.githubusercontent.com/20676176/98624424-25e4b200-2351-11eb-98b5-06dad1fa5b75.jpg)

**Comment by Ecofriendlyhooney** [comment]
[CPU speed]

Epoch 1/3
92/92 [==============================] - 93s 1s/step - loss: 1.4109 - accuracy: 0.3818 - val_loss: 1.1839 - val_accuracy: 0.4905
Epoch 2/3
92/92 [==============================] - 113s 1s/step - loss: 1.1072 - accuracy: 0.5473 - val_loss: 1.0622 - val_accuracy: 0.5845
Epoch 3/3
92/92 [==============================] - 104s 1s/step - loss: 0.9860 - accuracy: 0.6161 - val_loss: 0.9646 - val_accuracy: 0.6240
<tensorflow.python.keras.callbacks.History at 0x7f9315e330b8>
